### PR TITLE
[stable/postgresql] add persistence.existingClaim

### PR DESCRIPTION
**Pull request by flah00**
@prydonius Based on our conversation yesterday, I thought this might be a reasonable solution. Please let me know what you think. If this is agreeable, I can apply these basic changes to redis, rabbit, redmine, and jenkins.

### CODE REVIEW

**Comment by flah00** [comment]
I signed it!

File: stable/postgresql/templates/_helpers.tpl
Comment (databus23) [inline stable/postgresql/templates/_helpers.tpl:17]: I'm not sure creating this 7 line helper function is necessary in this case. It is only used once and the if statement can be inlined in the deployment in one line. (E.g. https://github.com/kubernetes/charts/blob/0e646ace488694573d921df8e4aa5ebb18dcca25/stable/prometheus/templates/server-deployment.yaml#L71). IMO this indirection costs more then it provides.

File: stable/postgresql/README.md
Comment (databus23) [inline stable/postgresql/README.md]: This documentation and example are for a very specific use case. I'm not sure but my feeling is that creation of physical volumes is outside the scope of this README. Im my mind this section should be shorter and more generic. The main purpose of the `existingClaim` option is to use an already exiting PVC and don't have helm create an new claim, this is generally useful to migrate exiting deployments to helm while keeping the data.

File: stable/postgresql/templates/_helpers.tpl
Comment (flah00) [reply stable/postgresql/templates/_helpers.tpl:17]: Yeah, I wasn't really sure which was more desirable, @prydonius and I briefly discussed the helper as a possible implementation... Though I could've easily misunderstood what he meant. 

I can unroll it.

File: stable/postgresql/README.md
Comment (flah00) [reply stable/postgresql/README.md]: Do you think I should remove the creation of the PV and PVC? Should the mention of NFS also be removed?

**Review by databus23 — APPROVED** [review]

**Comment by databus23** [comment]
LGTM

**Comment by flah00** [comment]
@databus23 Hey, is it possible that this PR requires the `lgtm` tag, rather than just the comment? I've noticed that  other PRs have the tag applied.

**Comment by databus23** [comment]
@flah00 Could be. I don't have committer access. ping @mgoodness

**Comment by flah00** [comment]
@databus23 @mgoodness I got some feedback on 394a4ef59df912964219e51850c497e852640ff1, do you want me to clean this up as well? Or are you happy with this PR as it is?

**Comment by prydonius** [comment]
@flah00 I updated with the other changes to be consistent, and bumped the minor version according to semver :)

**Pull request merged**
